services/notification/notifiers/status/base.py

```diff
@@ -336,26 +357,37 @@ async def send_notification(self, comparison: Comparison, payload):
                 "state": state,
                 "message": message,
             }
+
+            all_shas_to_notify = [head_commit_sha] + list(
+                comparison.context.gitlab_extra_shas or set()
```

Comment:
Optional: leave a comment explaining why we have to do this, also optional since you already left a comment in the gitlab_extra_shas definition

Reply:
I was hoping that having a comment both in the `gitlab_extra_shas` definition and the function that calculates those extra shas would suffice.